include/FreeRTOS.h

```diff
@@ -2402,15 +2402,15 @@
 #endif
 
 #ifndef traceENTER_xStreamBufferGenericCreate
-    #define traceENTER_xStreamBufferGenericCreate( xBufferSizeBytes, xTriggerLevelBytes, xIsMessageBuffer, pxSendCompletedCallback, pxReceiveCompletedCallback )
+    #define traceENTER_xStreamBufferGenericCreate( xBufferSizeBytes, xTriggerLevelBytes, xIsMessageBuffer, xIsBatchingBuffer, pxSendCompletedCallback, pxReceiveCompletedCallback )
```

Comment:
The following macros also need to be udpated:
* traceSTREAM_BUFFER_CREATE_FAILED
* traceSTREAM_BUFFER_CREATE_STATIC_FAILED
* traceSTREAM_BUFFER_CREATE